Utilities/GenerateDocs/doxy2swig.py

```diff
@@ -451,8 +451,36 @@ def __init__(self, src, javaFlag=0):
         self.FilterTitle = True
         self.sitkClassName=''
         self.EmptyText = False
+        # compiled regular expressions
+        # common formula types in xml version of documentation
         self.dollarFormula = re.compile("^\\$(.+)\\$$")
         self.arrayFormula = re.compile("^\\\\\\[(.+)\\\\\\]$")
+        # more complex formula layout, that breaks R documentation
+        # checks.
+        self.mathstuff1 = re.compile(r"\\begin\{array\}\{[^}]+\}")
+        self.mathstuff2 = re.compile(r"\\begin\{array\}")
+        self.mathstuff3 = re.compile(r"\\end\{array\}")
+        # a complex recursive regular expression, to deal with formula
+        # inside mbox and text structures
+        self.mathstuff4 = regex.compile(r"\\mbox({((?>[^}{]*(?1)?)*)})", flags=regex.V1)
+        self.mathstuff5 = regex.compile(r"\\text({((?>[^}{]*(?1)?)*)})", flags=regex.V1)
+        # the special doxygen tags - note - not greedy
+        self.mathstuff6 = re.compile(r"\\f\$(.+?)\\f\$")
+        # alignment tags
+        self.mathstuff7 = re.compile(r" & ")
+    def filterLatexMath(self, txt):
```

Comment:
PEP8: space before function definition. and 

"""
Comment about what the function is doing.
"""